src/transformers/models/albert/modeling_albert.py

```diff
@@ -57,6 +57,13 @@
 _CONFIG_FOR_DOC = "AlbertConfig"
 _TOKENIZER_FOR_DOC = "AlbertTokenizer"
 
+# SequenceClassification docstring
```

Comment:
Thanks a lot for taking care of all those models that's great! For all PyTorch models, could you maybe just add the correct values to the `@add_code_sample_docstring(...)` as is done here: https://github.com/huggingface/transformers/pull/16363/files#r833804669 - no need to overwrite the existing doc string normally :-)

Sorry we only changed to this format today, but in general there is not need to write new doc string code, one only needs to find the correct checkpoints and expected outputs as shown in the link above. 

I've explained here how one can find the good checkpoints: https://github.com/huggingface/transformers/issues/16292#issuecomment-1076919075

It would be amazing if you could do the same for all other models and TF models as well once: https://github.com/huggingface/transformers/pull/16370